sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -54,11 +56,27 @@ export class InteractiveBrowserCredential implements TokenCredential {
     this.persistenceEnabled = false;
     this.authenticationRecord = undefined;
 
+    if (options && options.redirectUri) {
+      if (typeof options.redirectUri === "string") {
+        this.redirectUri = options.redirectUri;
+      } else {
+        this.redirectUri = options.redirectUri();
+      }
+    } else {
+      this.redirectUri = "http://localhost";
+    }
+
+    if (options && options.authorityHost) {
+      this.authorityHost = options.authorityHost;
```

Comment:
Authority host will just have the host and protocol. For instance  `https://login.microsoftonline.com/`. We would need to append tenantId in both cases to get the full authority url.

Reply:
Ahh, missed that. Thanks. Should be fixed now.